package.json

```diff
@@ -52,5 +52,14 @@
     "typescript-eslint": "^7.8.0",
     "vega-datasets": "^2.8.0"
   },
-  "workspaces": ["packages/*"]
+  "workspaces": [
+    "packages/*"
+  ],
+  "engines": {
+    "node": ">=18.18.0"
+  },
+  "packageManager": "yarn@1.22.19",
+  "volta": {
```

Comment:
Recommended node version manager - can drop this if it's distracting / people prefer to bring their own node version management solution

https://volta.sh/

As it stands, you won't be able to build the project if you have an older version (like node 18.15) running locally.